.github/workflows/main.yml

```diff
@@ -86,6 +86,11 @@ jobs:
       run: yq -i 'del(.metadata.annotations."sonataflow.org/profile")' ${{ inputs.workflow_id }}/manifests/01-sonataflow*.yaml
     - name: Set container image ref in SonataFlow resource
       run: yq -i '.spec.podTemplate.container.image="quay.io/orchestrator/serverless-workflow-${{ inputs.workflow_id }}:${{ github.sha }}"' ${{ inputs.workflow_id }}/manifests/01-sonataflow*.yaml
+    - name: Set persistence spec in SonataFlow resource
```

Comment:
just asking: 
1. do we need a similar patch in serverless-workflows-config for kustomize deployments?
1. and in the Tekton pipeline triggered by software workflows?

Reply:
> and in the Tekton pipeline triggered by software workflows?

https://github.com/parodos-dev/orchestrator-helm-chart/pull/127

> do we need a similar patch in serverless-workflows-config for kustomize deployments?

Yes, good point